src/profile/sigmapr.jl

```diff
@@ -0,0 +1,68 @@
+"""
+    refitσ!(m::LinearMixedModel{T}, σ::T, tc::TableColumns{T}, obj::T, neg::Bool)
+
+Refit the model `m` with the given value of `σ` and return a NamedTuple of information about the fit.
+
+`obj` and `neg` allow for conversion of the objective to the `ζ` scale and `tc` is used to return a NamedTuple
+"""
+function refitσ!(
+    m::LinearMixedModel{T}, σ, tc::TableColumns{T}, obj::T, neg::Bool
+) where {T}
+    m.optsum.sigma = σ
+    refit!(m; progress=false)
+    return mkrow!(tc, m, (neg ? -one(T) : one(T)) * sqrt(m.objective - obj))
+end
+
+"""
+    _facsz(m, σ, objective)
+
+Return a factor such that refitting `m` with `σ` at its current value times this factor gives `ζ ≈ 0.5`
+"""
+function _facsz(m::LinearMixedModel{T}, σ::T, obj::T) where {T}
+    i64 = T(inv(64))
+    expi64 = exp(i64)     # help the compiler infer it is a constant
+    m.optsum.sigma = σ * expi64
+    return exp(i64 / (2 * sqrt(refit!(m; progress=false).objective - obj)))
+end
+
+"""
+    profileσ(m::LinearMixedModel, tc::TableColumns; threshold=4)
+
+Return a Table of the profile of `σ` for model `m`.  The profile extends to where the magnitude of ζ exceeds `threshold`.
+"""
```

Comment:
```suggestion
!!! note
   This method is called by `profile` and currently considered internal.
   As such, it may change or disappear in a future release without being considered breaking.
   
"""
```